tests/Common/FilesTest.php

```diff
@@ -24,6 +24,7 @@ public function testFileList()
         $uploadedFile = reset($files);
         $this->assertEquals($fileId, $uploadedFile['id']);
         $this->assertArrayHasKey('region', $uploadedFile);
+        $this->assertArrayNotHasKey('credentials', $uploadedFile);
```

Comment:
Tady ale netestuješ to co píšeš v zadání:) Tenhle test bych nechal ale ještě přidal test na to kde byl ten bug tzn. že getFile bez federation token nevrací ty credentials.

Reply:
Nejlepší je postupovat v případě bugu:
 - failing test
 - fix na backendu

Reply:
Máš pravdu.

Přidal jsem test na `get`. Myslel jsem si, že list a get bude využívat stejné fce, ale není tomu tak. Díky! :+1: